### PR TITLE
6611 change all the rollbar calls to use our log wrapper

### LINKED ISSUE #6611: Change all the Rollbar calls to use our Log wrapper
We are calling directly Rollbar in many points of the platform and we should change them to use our logging wrapper https://github.com/CartoDB/cartodb/blob/master/config/initializers/error_notifier.rb#L16-L54

### PR DESCRIPTION
**Pull request by juanignaciosl**
This closes #6611 and #6752.

### CODE REVIEW

**Comment by juanignaciosl** [comment]
@gfiorav CR this, please. @rafatower , could you take a look at this as well, please? #6611 was labeled for data-services but I didn't realize when I begun with #6752. I've done this with two commits, being the second the one that most affects you.

**Comment by rafatower** [comment]
:+1:

**Comment by juanignaciosl** [comment]
@gfiorav last commit tamed the hound, last CR!

**Comment by juanignaciosl** [comment]
@javitonino 👀 , please, as @gfiorav is out.

File: services/datasources/lib/datasources/search/twitter.rb [hunk 1]
```diff
@@ -226,17 +225,6 @@ def get_audit_stats
           { :retrieved_items => entry.retrieved_items }
         end
 
-        # Sets an error reporting component
-        # @param component mixed
-        # @throws DatasourceBaseError
-        def report_component=(component)
```
Comment (javitonino) [inline services/datasources/lib/datasources/search/twitter.rb]: Wow! Good riddance!

**Pull request merged**